modules/artic/minion/main.nf

```diff
@@ -43,7 +43,9 @@ process ARTIC_MINION {
         summary = ""
         model = file(medaka_model).exists() ? "--medaka-model ./$medaka_model" : "--medaka-model $medaka_model"
     }
+    def hd5_plugin_path = params.options.hd5_plugin_path ? "HDF5_PLUGIN_PATH=" + params.options.hd5_plugin_path : "HDF5_PLUGIN_PATH=/usr/local/lib/python3.6/site-packages/ont_fast5_api/vbz_plugin"
     """
+    $hd5_plugin_path \\
```

Comment:
```suggestion
    def hd5_plugin_path = params.options.hd5_plugin_path ? "HDF5_PLUGIN_PATH=" + params.options.hd5_plugin_path : "HDF5_PLUGIN_PATH=/usr/local/lib/python3.6/site-packages/ont_fast5_api/vbz_plugin"
    """
    HDF5_PLUGIN_PATH=/usr/local/lib/python3.6/site-packages/ont_fast5_api/vbz_plugin

```

Reply:
I wonder whether we just hard-code it here? If it is always shipped with the container then the main issue is making sure that path exists in the container when it is updated. Can we add another test for this somehow?

Reply:
Would the path above be different if using Conda or a Container in which case the suggestion above won't work.